pkg/cmd/bootstrap.go

```diff
@@ -149,7 +162,7 @@ func missingFlagErr(flags []string) error {
 func initiateInteractiveMode(io *BootstrapParameters, client *utility.Client, cmd *cobra.Command) error {
 	log.Progressf("\nStarting interactive prompt\n")
 	// Prompt if user wants to use all default values and only be prompted with required or other necessary questions
-	promptForAll := !ui.UseDefaultValues()
+	promptForAll := !ui.UseDefaultValues(defaultFlagValues)
```

Comment:
Since we no longer use `defaultFlagValues` map in the bootstrap package, can we directly move it inside the function `UseDefaultValues()`?